src/main/java/org/spongepowered/common/mixin/core/server/MixinNetHandlerPlayServer.java

```diff
@@ -187,4 +193,32 @@ public void processCommandBlock(C17PacketCustomPayload packetIn, CallbackInfo ci
 
     }
 
+    @Inject(method = "processPlayer", at = @At(value = "FIELD", target = "net/minecraft/network/NetHandlerPlayServer.hasMoved", ordinal = 2), cancellable = true, remap = false)
```

Comment:
I honestly don't understand the remap thing of mixins. Only thing I understood of it, is that it will make updating sponge easier.